db/migrate/20240219110029_add_expiration_date_to_group_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddExpirationDateToGroupUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column :group_users, :to_expire_on, :datetime
```

Comment:
should we also change this to `expiration_date`?

Reply:
Updated to have only 1 migration file, removed others